Wallet user update

waveform06 opened the pull request:
Post solana phrasing with updates and added info on how to identify migrated wallet.

waveform06 committed:
Wallet user update

Post solana phrasing with updates and added info on how to identify migrated wallet.

waveform06 commented:
Actually helium hotspot app needs to be 2.0.8 or higher

jthiller commented on docs/architecture/solana/migration/wallet-user.mdx:
```diff
@@ -9,24 +9,23 @@ slug: /solana/migration/wallet-user
 
 You can migrate a wallet using the widget below by entering a Helium or Solana wallet address and
-clicking the "Seed Wallet" buttton.
+clicking the "Seed Wallet" buttton. Wallets seeded/migrated will have 0.00139088 SOL deposited in
```
typo, "buttton"

jthiller commented on docs/architecture/solana/migration/wallet-user.mdx:
```diff
@@ -38,13 +37,14 @@ import { MigrateWallet } from '@theme/MigrateWallet'
 
 The Helium Hotspot App is no longer used to manage Tokens or perform Wallet activities and instead
 is only used to manage Original Helium Hotspots and FreedomFi Hotspots.
+Hotspots are now managed with Maker Apps or the Helium Wallet.
```
Should downplay maker apps as we're working to bring these features back to Helium Wallet.

jthiller commented on docs/architecture/solana/migration/wallet-user.mdx:
```diff
@@ -38,13 +37,14 @@ import { MigrateWallet } from '@theme/MigrateWallet'
-To start, ensure your Helium Hotspot app is updated to version **5.0.0** or later and that you have
-downloaded the latest Helium Wallet app version **2.0.0** or higher.
+To start, ensure your Helium Hotspot app is updated to at least version **5.0.0** or later and that 
+the Helium Wallet app version **2.0.0** or later. The latest versions of each app are recommended.
```
"The latest versions of each app are recommended." not needed

waveform06 committed:
Update wallet-user.mdx after review

We ought to keep in some reference to maker apps as some of them have additional features the Helium apps do not have.

jthiller committed:
linting and some cleanup
